database-postgresql/schema.sql

```diff
@@ -0,0 +1,328 @@
+DROP SCHEMA public CASCADE;
+
+CREATE SCHEMA public;
+
+CREATE TABLE trailPhotos (
+  photo_id SERIAL PRIMARY KEY,
+  trail_id integer,
+  user_id integer,
+  upload_date timestamptz,
+  photo_url text,
+  caption text,
+  is_hero_photo boolean
+);
+
+-- DO NOT EDIT LINES ABOVE, PSQL INSERTION STATEMENTS BELOW
```

Comment:
Not high priority, however, a schema.sql file should just schema. Below this line is seeding. 

Possible to have your seed generation file pipe into another function (or script) which will seed your db.